.github/workflows/check.yaml

```diff
@@ -21,3 +22,7 @@ jobs:
       - run: make setup
       - run: make check
 
+      - name: Install dependencies for webapp
+        run: cd packages/webapp && pnpm install
+      - name: Check formatting with Prettier
+        run: cd packages/webapp && npx prettier --check "**/*.{js,jsx,ts,tsx,json,css}"
```

Comment:
This should be folded into `make check` in the webapp Makefile (which will cause it be checked by the global Makefile `make check` and thus covered by the above)